voting/src/voting.rs

```diff
@@ -139,7 +139,24 @@ impl TxOwner for Context {
     }
 
     fn check_transaction(&self, transaction: &Transaction) -> Result<(), coordinator::types::ErrorCode> {
-        unimplemented!();
+        let todo_fixthis: coordinator::types::ErrorCode = 3;
+        match transaction.tx_type() {
+            CREATE_VOTE_PAPER_TX_TYPE => {
+                let tx: CreateVotePaperTransaction =
+                    serde_cbor::from_slice(&transaction.body()).map_err(|_| todo_fixthis)?;
+                tx.verify().map_err(|_| todo_fixthis)?;
+                let meeting_id = tx.tx.action.general_meeting_id;
+                let agendas = tx.tx.action.agenda_number;
```

Comment:
Please check your code that it has the same problem as before.

https://github.com/CodeChain-io/foundry/pull/466#discussion_r469675774

> The name `agendas` looks not a good name.
> `agenda_number` is a number of agenda. If there are 5 agendas in a general meeting, there could be 5 vote papers for each agenda.
> `agendas` looks like there are multiple agendas in a voting paper. However, it is not true.